Description: Avoid name duplication of 'cdevname'.
 The identifier 'cdevname' exists in 'sbin/fsck/fsutils.*'
 as well as in 'sbin/fsck_ffs/fsutil.*'. Hide the former
 occurrences.
Author: Petr Salinger <petr.salinger@seznam.cz>
Forwarded: not-needed
Last-Update: 2011-03-01

--- dragonflybsd-quota-8.2.orig/sbin/fsck/fsutil.c
+++ dragonflybsd-quota-8.2/sbin/fsck/fsutil.c
@@ -64,11 +64,13 @@ setcdevname(const char *cd, int pr)
 	preen = pr;
 }
 
+#if 0
 const char *
 cdevname(void)
 {
 	return dev;
 }
+#endif
 
 static void
 vmsg(int fatal, const char *fmt, va_list ap)
--- dragonflybsd-quota-8.2.orig/sbin/fsck/fsutil.h
+++ dragonflybsd-quota-8.2/sbin/fsck/fsutil.h
@@ -39,7 +39,9 @@ void pwarn(const char *, ...)
 void panic(const char *, ...) __attribute__ ((noreturn))
      __THROW __attribute__ ((__format__ (__printf__, 1, 2))) __wur;
 const char *devcheck(const char *);
+#if 0
 const char *cdevname(void);
+#endif
 void setcdevname(const char *, int);
 struct statfs *getmntpt(const char *);
 void *emalloc(size_t);
